.github/workflows/test-unit.yml

```diff
@@ -25,16 +25,16 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - name: npm ci
-        run: npm ci
+        run: npm ci --legacy-peer-deps
```

Comment:
Ditto here?

Reply:
The parameter `--legacy-peer-deps` ensures that if there are any conflicts in packages, they are resolved instead of failing.